com.github.mtkennerly.ludusavi.yaml

```
@@ -0,0 +1,52 @@
+app-id: com.github.mtkennerly.ludusavi
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: /run/build/ludusavi/cargo
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
```

does it really support wayland?

either way
```suggestion
  - --socket=fallback-x11
  - --socket=wayland
  - --share=ipc
```

IPC is still necessary.

---

I havent tested it (use x11 bc nvidia), but it uses iced.rs, so it should work.

---

It does work on Wayland. For example, some users have verified that it works using the Steam Deck's game mode, which runs Wayland.